src/content/developers/docs/consensus-mechanisms/pos/gasper/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Gasper
+description: An explanation of the Gasper proof-of-stake mechanism.
+lang: en
+sidebar: true
+incomplete: false
+---
+
+Gasper is a combination of Casper the Friendly Finality Gadget (Casper-FGG) and the LMD-GHOST fork choice algorithm. Together these components form the consensus mechanism securing proof-of-stake Ethereum. Casper is the mechanism that upgrades certain blocks to "finalized" so that new entrants into the network can be confident that they are syncing the canonical chain. The fork choice algorithm uses accumulated votes to ensure that nodes can easily select the correct one when forks arise in the blockchain.
+
+**note** that the original definition of Casper-FFG was updated slightly for inclusion in Gasper. On this page we consider the updated version.
+
+## The role of Gasper {#role-of-gasper}
+
+Gasper sits on top of a proof-of-stake blockchain where nodes provide ether as a security deposit that can be destroyed if they are lazy or dishonest in proposing or validating blocks. Gasper is the mechanism defining how validators get rewarded and punished, decide which blocks to accept and reject, and which fork of the blockchain to build on.
+
+## What is finality? {#what-is-finality}
+
+[Casper the Friendly Finality Gadget (Casper-FFG)](https://arxiv.org/pdf/1710.09437.pdf) is an algorithm that finalizes blocks. Finalizing a block means upgrading a block so that it cannot be reverted (unless there has been a critical consensus failure). Finalized blocks can be thought of as information the blockchain is certain about. A block must pass through a two-step upgrade procedure for a block to be finalized:
```

Comment:
Small note, but feel like we should open this section with a sentence directly answering "What is finality?" before explaining more about Casper, etc.

Reply:
yup - done